test/NuGet.Core.Tests/NuGet.ProjectModel.Test/JsonPackageSpecReaderTests.cs

```diff
@@ -46,6 +47,45 @@ public void PackageSpecReader_PackageMissingVersion()
         }
 
         [Fact]
+        public void PackageSpecReader_ReadImports()
```

Comment:
Now that you commented, I remember we need to add an equivalent test to dev...doing that now.